x-pack/legacy/plugins/alerting/README.md

```diff
@@ -198,6 +198,7 @@ Payload:
 |Property|Description|Type|
 |---|---|---|
 |enabled|Indicate if you want the alert to start executing on an interval basis after it has been created.|boolean| 
+|tags|A list of tags to reference and search in the future.|string[]|
```

Comment:
Feels like this description is not quite right.  Just looked at ECS's definition - "List of keywords used to tag each event."  mkay heh.  But perhaps "keywords" would be useful as "A list of keywords ...".

Reply:
Fixed in https://github.com/elastic/kibana/pull/49320/commits/356e2e6e62da332c1a6490f9d52572ee97f6dbf4.